readme.md

```
@@ -287,6 +287,22 @@ Note: Command-line flags will override any values set in environment variables o
         1. run `.\whatsapp.exe --help` for more detail flags
 6. open `http://localhost:3000` in browser
 
+### Cross-Compile for Raspberry Pi (ARM)
+
+If you want to build for Raspberry Pi or other ARM devices without needing a C toolchain (CGO), you can use the `purego` build tag. This will use a pure-Go SQLite implementation.
+
+1. **Raspberry Pi Zero / 1 (ARMv6):**
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)